src/mobility/components/sheets/ScooterSheet.tsx

```diff
@@ -35,6 +35,7 @@ type Props = {
   navigateSupportCallback: () => void;
   loginCallback: () => void;
   startOnboardingCallback: () => void;
+  navigation: any;
```

Comment:
Can you specify type here?
```suggestion
  navigation: RootNavigationProps;
```

Reply:
yes! updated